test/parallel/test-debugger-util-regression.js

```diff
@@ -21,12 +23,16 @@ proc.stderr.setEncoding('utf8');
 
 let stdout = '';
 let stderr = '';
+proc.stdout.on('data', (data) => stdout += data);
+proc.stderr.on('data', (data) => stderr += data);
```

Comment:
I think `proc.stdout.setEncoding('utf-8')` would be good practice here (and the same for `proc.stderr`)